pkg/triton/triton.go

```diff
@@ -73,7 +75,15 @@ func NewTriton() Triton {
 func (ts *triton) Init() {
 	grpcUri := config.Config.TritonServer.GrpcURI
 	// Connect to gRPC server
-	conn, err := grpc.Dial(grpcUri, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	maxMsgSize := 1024 * 1024 * 20 // 20 MB
```

Comment:
avoid hardcoding, adjust [this](https://github.com/instill-ai/model-backend/blob/main/config/config.yaml#L16) and use it as base unit here and determine how large total message size should be

Reply:
Got it, I have made a change which moving the message size setting part into conf.go